drivers/storage/portworx/component/csi.go

```diff
@@ -563,6 +563,7 @@ func getCSIDeploymentSpec(
 			"--csi-address=$(ADDRESS)",
 			"--leader-election=true",
 			"--default-fstype=ext4",
+			"--extra-create-metadata=true",
```

Comment:
provisioner v2+ is not covered in unit tests, we may want to add that as the default in the future

Reply:
Ok yes we can add that in the future